src/components/PlaylistPage/LessonItem.js

```diff
@@ -20,22 +23,31 @@ export const LessonItem = React.createClass({
         </Button>
       </LinkContainer>
       : null;
+    const tooltipContent = getLessonIntro(lesson.path.slice(1));
+    const tooltip =
+      <Tooltip id="lessonIntro">
+        <div dangerouslySetInnerHTML={{__html: tooltipContent}} />
```

Comment:
Ikke lag objectet `{__html: ...}` rett i `dangerouslySetInnerHTML`: Se https://facebook.github.io/react/docs/dom-elements.html#dangerouslysetinnerhtml

Se forøvrig hvordan det er gjort i Lesson.js. Poenget er at funksjonen createMarkup (eller hva nå man vil kalle funksjonen) har ansvaret for å levere "trygg" html, og at man ikke ved et feiltakelse (eller ondsinnet hacking) får inn skummel html.